packages/mcp-provider-dx-core/tsconfig.json

```diff
@@ -9,9 +9,7 @@
   },
   "include": ["./src"],
   "references": [
-    {
-      "path": "../mcp-provider-api/tsconfig.json",
-      "path": "../mcp-test-client/tsconfig.json"
-    }
```

Comment:
each reference needs to be in its own object, see:
https://www.typescriptlang.org/docs/handbook/project-references.html